.storybook/preview.js

```diff
@@ -1,6 +1,7 @@
 import { initialize, mswDecorator } from "msw-storybook-addon";
```

Comment:
`msw-storybook-addon` and `handlers` are needed for enabling mock handlers in StoryBook.

The `handlers` placed into separate module for sharing between StoryBook and unit tests